tests/e2e/test_pvc_creation_performance.py

```diff
@@ -43,9 +46,50 @@ def test_pvc_creation_measurement_performance(self, teardown_factory):
 
         pvc_obj = create_pvc(sc_name=self.sc_obj.name, size=self.pvc_size)
         teardown_factory(pvc_obj)
-        create_time = measure_pvc_creation_time('CephBlockPool', pvc_obj.name)
+        create_time = measure_pvc_creation_time(self.interface, pvc_obj.name)
         if create_time > 1:
             raise AssertionError(
                 f"PVC creation time is {create_time} and greater than 1 second"
             )
         logging.info("PVC creation took less than a 1 second")
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    @polarion_id('OCS-1620')
+    @bugzilla('1741612')
+    def test_multiple_pvc_creation_measurement_performance(
+        self, teardown_factory
+    ):
+        """
+        Measuring PVC creation time of 120 PVCs in 60 seconds
+        """
+        number_of_pvcs = 120
+        log.info('Start creating new 120 PVCs')
+
+        pvc_objs = create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=self.pvc_size,
+            wait=False
+        )
+        for pvc_obj in pvc_objs:
+            teardown_factory(pvc_obj)
+        for pvc_obj in pvc_objs:
+            wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
+        start_time = measure_pvc_creation_time(
+            self.interface, pvc_objs[0].name, return_start_time=True
+        )
+        end_time = measure_pvc_creation_time(
+            self.interface, pvc_objs[number_of_pvcs - 1].name,
+            return_end_time=True
+        )
+        total = end_time - start_time
+        total_time = total.total_seconds()
+        if total_time > 60:
```

Comment:
I think an explicit assertion would be clearer here, but otherwise LGTM.
```python
assert total_time <= 60, f"{number_of_pvcs} PVCs creation time is ..."
```